leadbutt.py

```diff
@@ -179,13 +357,11 @@ def get_metric_statistics(**kwargs):
         period_local = options['Period'] * 60
         count_local = options['Count']
         # if you have metrics that are available only every 5 minutes, be sure to request only stats
-        # that are likely/sure to be up to date, ie ones ending on the previous period increment.
```

Comment:
I prefer the previous formatting, it made clearer the difference between how the two times are calculated, can you revert this whitespace change?